src/fsi/src/partitioned/4C_fsi_partitioned.cpp

```diff
@@ -678,20 +679,20 @@ bool FSI::Partitioned::computeF(const Epetra_Vector& x, Epetra_Vector& F, const
   if (Core::Communication::my_mpi_rank(get_comm()) == 0)
   {
     utils_->out() << "\n " << "FSI residual calculation" << ".\n";
-    if (fillFlag != Residual) utils_->out() << " fillFlag = " << flags[fillFlag] << "\n";
+    if (fill_flag != Residual) utils_->out() << " fillFlag = " << flags[fill_flag] << "\n";
   }
 
   // we count the number of times the residuum is build
-  counter_[fillFlag] += 1;
+  counter_[fill_flag] += 1;
```

Comment:
The suggestion of @amgebauer will require to redeclare `counter_` as `std::map`. An alternative would be to define a helper for explicit conversion of the enum, but I it is quite ugly (smth like `return static_cast<typename std::underlying_type<E>::type>(e)`) and does not reflect the purpose of `counter_`.